wiki/en/en-Tips-Tricks-More.md

```diff
@@ -96,12 +94,10 @@ jack_connect Jamulus:'output left' system:playback_1
 jack_connect Jamulus:'output right' system:playback_2
 ~~~
 
-
-
 ## Using ctrlmidich for MIDI controllers
 
 The volume faders in the client's mixer window can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There are two parameters you can set: `Channel` and `Offset`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and the second parameter to the Control Number you want the first fader to react to. By default, the offset is 70 (for the Behringer X-Touch), which means that the first fader reacts to Control Number 70, the second to 71, and so on.
 
 So for example, if you're using a Behringer X-Touch, sending MIDI on channel 1 and leaving the offset at default, the command would look like this: `--ctrlmidich 1`. If you have a different controller, e.g. sending MIDI on channel 2 and starting with Control Number 30, the command would be as follows: `--ctrlmidich "2;30"`
 
-Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (Qjackctl (Linux), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in Qjackctl.
+Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (QJackCtl (Linux), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in QJackCtl.
```

Comment:
I thought this was the right spelling (what you corrected it to - I used that in my translation), but after checking its Github page, it in fact appears to be spelled 'QjackCtl'.

Reply:
Ok. If that's the case, we must spell it like that everywhere. 

But I assume you will also look through the wiki and fix some other spelling mistakes?

Reply:
That was my intention, a lot of the stuff you've highlighted here could go in now but there are things I don't entirely agree with, which I'll comment on - and maybe @gilgongo will chip in too.

Reply:
Yes. Let's see this PR as a draft for possible typo fixes and not merge it (since it has the wrong base)

Reply:
Went through the wiki with search & replace and replaced some (should be on changes now)